[DOCS] Adds the 8.0.0-beta1 release notes

KOTungseth opened the pull request:
## Summary

Adds the [release notes for the 8.0.0-beta1 release](https://kibana_116483.docs-preview.app.elstc.co/guide/en/kibana/master/release-notes-8.0.0-beta1.html).

lukeelmers commented on docs/CHANGELOG.asciidoc:
```diff
@@ -10,10 +10,496 @@
+Support for the following configuration settings has been removed:
+
+* `newsfeed.defaultLanguage`
+* `xpack.banners.placement`
```
Unlike other items in this list, support for this specific config has not actually been removed, just one of the values it accepts has changed (as explained in the details below). 

We might want to relocate this item elsewhere to prevent confusion.

lukeelmers commented on docs/CHANGELOG.asciidoc:
```diff
@@ -10,10 +10,496 @@
+
+*Impact* +
+* `newsfeed.defaultLanguage` -- Newsfeed items are retrieved based on the browser locale and default to English
+* `xpack.banners.placement` has changed value from 'header' to 'top'
```
nit: I might rephrase this to something like "The 'header' value provided to the xpack.banners.placement config has been renamed to 'top'"

lukeelmers commented on docs/CHANGELOG.asciidoc:
```diff
@@ -10,10 +10,496 @@
+[%collapsible]
+====
+*Details* +
+The `so/server/es` settings have been removed. If your configuration uses the following settings, {kib} fails to start:
```
```suggestion
Some of the `so/server/es` settings have been removed. If your configuration uses the following settings, {kib} fails to start:
```

lukeelmers commented on docs/CHANGELOG.asciidoc:
```diff
@@ -10,10 +10,496 @@
+The `kibana.index`, `xpack.reporting.index`, and `xpack.task_manager.index` settings have been removed. For more information, refer to {kibana-pull}112773[#112773].
+
+*Impact* +
+Use spaces, cross-cluster replication, or cross-cluster search. To migrate to <<xpack-spaces,spaces>>, export your <<managing-saved-objects, saved objects>> from a tenant into the default tenant in a space. For more details, refer to link:https://github.com/elastic/kibana/issues/82020[#82020].
```
I'm not sure I understand the phrasing "default tenant in a space" -- I'd recommend removing it

```suggestion
Use spaces, cross-cluster replication, or cross-cluster search. To migrate to <<xpack-spaces,spaces>>, export your <<managing-saved-objects, saved objects>> from a tenant into the default space. For more details, refer to link:https://github.com/elastic/kibana/issues/82020[#82020].
```

jportner reviewed (commented):
Sorry for the late review on behalf of Platform Security.
All of our breaking changes in 8.0 are listed here: #111160
(you can ignore the ones that are marked as "Future", we have deprecated those but they are not breaking in 8.0)

I see in these 8.0.0-beta1 release notes you noted:
* Remove legacy audit logging
* Prevent disabling the security plugin (in "Removes `xpack.security.enabled` setting")
* Prevent disabling the spaces plugin (in "Removes `enabled` settings from plugins")
* Prevent disabling the encryptedSavedObjects plugin (in "Removes `enabled` settings from plugins")
* Include ML feature in base privileges

I see in the 8.0.0-alpha2 release notes you noted:
* Remove kibana_dashboard_only_user role in favor of Kibana feature privileges

I did not see any mention of:
* Change Cases sub-feature to be a top-level feature (though maybe this should be in the "Elastic Security Solution Release Notes" instead?)

Also, we had a release note for "Add defaults for xpack.security.session.{lifespan,idleTimeout}" in [8.0.0-alpha1](https://kibana_116483.docs-preview.app.elstc.co/guide/en/kibana/master/release-notes-8.0.0-alpha1.html#settings-changes-8.0.0-alpha1), but that has now changed a bit. The new default for `xpack.security.session.idleTimeout` is now 8h instead of 1h. Should we updated it in the 8.0.0-beta1 release notes?

kibanamachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/3302)
* Commit: 1d8eefce37fb34c12338215eda5c4ca49588e85b

### History
* :green_heart: [Build #2299](https://buildkite.com/elastic/kibana-pull-request/builds/2299) succeeded 47917cca04756aa067a91009aa68008727723e14
* :green_heart: [Build #2298](https://buildkite.com/elastic/kibana-pull-request/builds/2298) succeeded 0e40406d42b5000cddd73567b6b4f69dce72c01e
* :green_heart: [Build #2185](https://buildkite.com/elastic/kibana-pull-request/builds/2185) succeeded 4879b4061075f914292eba56f6331b79b3355474

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @KOTungseth

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3302","number":3302,"commit":"1d8eefce37fb34c12338215eda5c4ca49588e85b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/2299","number":2299,"commit":"47917cca04756aa067a91009aa68008727723e14"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/2298","number":2298,"commit":"0e40406d42b5000cddd73567b6b4f69dce72c01e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/2185","number":2185,"commit":"4879b4061075f914292eba56f6331b79b3355474"}],"number":3302}
buildkite-pr-comment-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/3305)
* Commit: 2b819c63e99c65e64e6f572231a18b7e95128647

### History
* :green_heart: [Build #3302](https://buildkite.com/elastic/kibana-pull-request/builds/3302) succeeded 1d8eefce37fb34c12338215eda5c4ca49588e85b
* :green_heart: [Build #2299](https://buildkite.com/elastic/kibana-pull-request/builds/2299) succeeded 47917cca04756aa067a91009aa68008727723e14
* :green_heart: [Build #2298](https://buildkite.com/elastic/kibana-pull-request/builds/2298) succeeded 0e40406d42b5000cddd73567b6b4f69dce72c01e
* :green_heart: [Build #2185](https://buildkite.com/elastic/kibana-pull-request/builds/2185) succeeded 4879b4061075f914292eba56f6331b79b3355474

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @KOTungseth

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3305","number":3305,"commit":"2b819c63e99c65e64e6f572231a18b7e95128647"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3302","number":3302,"commit":"1d8eefce37fb34c12338215eda5c4ca49588e85b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/2299","number":2299,"commit":"47917cca04756aa067a91009aa68008727723e14"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/2298","number":2298,"commit":"0e40406d42b5000cddd73567b6b4f69dce72c01e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/2185","number":2185,"commit":"4879b4061075f914292eba56f6331b79b3355474"}],"number":3305}
buildkite-pr-comment-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/3324)
* Commit: dc45cc338c9456d19cb9a6b5143832a684295d5e

### History
* :green_heart: [Build #3305](https://buildkite.com/elastic/kibana-pull-request/builds/3305) succeeded 2b819c63e99c65e64e6f572231a18b7e95128647
* :green_heart: [Build #3302](https://buildkite.com/elastic/kibana-pull-request/builds/3302) succeeded 1d8eefce37fb34c12338215eda5c4ca49588e85b
* :green_heart: [Build #2299](https://buildkite.com/elastic/kibana-pull-request/builds/2299) succeeded 47917cca04756aa067a91009aa68008727723e14
* :green_heart: [Build #2298](https://buildkite.com/elastic/kibana-pull-request/builds/2298) succeeded 0e40406d42b5000cddd73567b6b4f69dce72c01e
* :green_heart: [Build #2185](https://buildkite.com/elastic/kibana-pull-request/builds/2185) succeeded 4879b4061075f914292eba56f6331b79b3355474

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @KOTungseth

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3324","number":3324,"commit":"dc45cc338c9456d19cb9a6b5143832a684295d5e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3305","number":3305,"commit":"2b819c63e99c65e64e6f572231a18b7e95128647"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3302","number":3302,"commit":"1d8eefce37fb34c12338215eda5c4ca49588e85b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/2299","number":2299,"commit":"47917cca04756aa067a91009aa68008727723e14"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/2298","number":2298,"commit":"0e40406d42b5000cddd73567b6b4f69dce72c01e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/2185","number":2185,"commit":"4879b4061075f914292eba56f6331b79b3355474"}],"number":3324}
buildkite-pr-comment-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/3414)
* Commit: 7b1c651262e0fdcd5922d10d60032ba523eeed03

### History
* :green_heart: [Build #3324](https://buildkite.com/elastic/kibana-pull-request/builds/3324) succeeded dc45cc338c9456d19cb9a6b5143832a684295d5e
* :green_heart: [Build #3305](https://buildkite.com/elastic/kibana-pull-request/builds/3305) succeeded 2b819c63e99c65e64e6f572231a18b7e95128647
* :green_heart: [Build #3302](https://buildkite.com/elastic/kibana-pull-request/builds/3302) succeeded 1d8eefce37fb34c12338215eda5c4ca49588e85b
* :green_heart: [Build #2299](https://buildkite.com/elastic/kibana-pull-request/builds/2299) succeeded 47917cca04756aa067a91009aa68008727723e14
* :green_heart: [Build #2298](https://buildkite.com/elastic/kibana-pull-request/builds/2298) succeeded 0e40406d42b5000cddd73567b6b4f69dce72c01e
* :green_heart: [Build #2185](https://buildkite.com/elastic/kibana-pull-request/builds/2185) succeeded 4879b4061075f914292eba56f6331b79b3355474

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @KOTungseth

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3414","number":3414,"commit":"7b1c651262e0fdcd5922d10d60032ba523eeed03"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3324","number":3324,"commit":"dc45cc338c9456d19cb9a6b5143832a684295d5e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3305","number":3305,"commit":"2b819c63e99c65e64e6f572231a18b7e95128647"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3302","number":3302,"commit":"1d8eefce37fb34c12338215eda5c4ca49588e85b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/2299","number":2299,"commit":"47917cca04756aa067a91009aa68008727723e14"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/2298","number":2298,"commit":"0e40406d42b5000cddd73567b6b4f69dce72c01e"}],"number":3414}
buildkite-pr-comment-->

KOTungseth commented:
> I did not see any mention of:
> 
> * Change Cases sub-feature to be a top-level feature (though maybe this should be in the "Elastic Security Solution Release Notes" instead?)

This should be documented in the Elastic Security Solution Release Notes.

> 
> Also, we had a release note for "Add defaults for xpack.security.session.{lifespan,idleTimeout}" in [8.0.0-alpha1](https://kibana_116483.docs-preview.app.elstc.co/guide/en/kibana/master/release-notes-8.0.0-alpha1.html#settings-changes-8.0.0-alpha1), but that has now changed a bit. The new default for `xpack.security.session.idleTimeout` is now 8h instead of 1h. Should we updated it in the 8.0.0-beta1 release notes?

I can make this change in a separate PR.

KOTungseth commented on docs/CHANGELOG.asciidoc:
```diff
@@ -10,10 +10,496 @@
+To audit {kib} events, enable the <<xpack-security-ecs-audit-logging, ECS audit logger>>.
+====
+
+[discrete]
+[[breaking-111681]]
+.Removes `xpack.security.enabled` setting
+[%collapsible]
+====
+*Details* +
+The `xpack.security.enabled` setting has been removed. For more information, refer to {kibana-pull}111681[#111681].
+
+*Impact* +
+Use {ref}/security-settings.html#general-security-settings[`xpack.security.enabled`] in elasticsearch.yml.
+====
```
@jportner `Removes xpack.security.enabled setting` is documented in a separate section is because the work was completed in #111681, and `Removes enabled settings from plugins` was completed in #113495. I can combine them if needed.

jportner replied:
Yeah, we only completed it in a separate PR because we had some complex test changes to make as a result.

I'd personally lean towards a single section since that's more concise and the release notes are already massive.

KOTungseth replied:
Done! Can you take a final look to make sure it still makes sense?

jportner replied:
Looks good

kibanamachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/3425)
* Commit: 77d4407b1cb8a15ef9a55b1973bced27dd222fcc

### History
* :green_heart: [Build #3414](https://buildkite.com/elastic/kibana-pull-request/builds/3414) succeeded 7b1c651262e0fdcd5922d10d60032ba523eeed03
* :green_heart: [Build #3324](https://buildkite.com/elastic/kibana-pull-request/builds/3324) succeeded dc45cc338c9456d19cb9a6b5143832a684295d5e
* :green_heart: [Build #3305](https://buildkite.com/elastic/kibana-pull-request/builds/3305) succeeded 2b819c63e99c65e64e6f572231a18b7e95128647
* :green_heart: [Build #3302](https://buildkite.com/elastic/kibana-pull-request/builds/3302) succeeded 1d8eefce37fb34c12338215eda5c4ca49588e85b
* :green_heart: [Build #2299](https://buildkite.com/elastic/kibana-pull-request/builds/2299) succeeded 47917cca04756aa067a91009aa68008727723e14
* :green_heart: [Build #2298](https://buildkite.com/elastic/kibana-pull-request/builds/2298) succeeded 0e40406d42b5000cddd73567b6b4f69dce72c01e

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @KOTungseth

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3425","number":3425,"commit":"77d4407b1cb8a15ef9a55b1973bced27dd222fcc"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3414","number":3414,"commit":"7b1c651262e0fdcd5922d10d60032ba523eeed03"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3324","number":3324,"commit":"dc45cc338c9456d19cb9a6b5143832a684295d5e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3305","number":3305,"commit":"2b819c63e99c65e64e6f572231a18b7e95128647"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/3302","number":3302,"commit":"1d8eefce37fb34c12338215eda5c4ca49588e85b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/2299","number":2299,"commit":"47917cca04756aa067a91009aa68008727723e14"}],"number":3425}
buildkite-pr-comment-->
